lib/mp4/transmuxer.js

```diff
@@ -546,104 +547,170 @@ Transmuxer = function(options) {
     videoTrack,
     audioTrack,
     packetStream, parseStream, elementaryStream,
-    aacStream, h264Stream,
+    adtsStream, h264Stream,aacStream,
     videoSegmentStream, audioSegmentStream, captionStream,
-    coalesceStream;
+    coalesceStream,
+    aacfile;
+
+  this.setupAACPipeline = function() {
```

Comment:
Our caps standard for this project means that this should be called `setupAacPipeline`.